packages/edit-site/src/components/save-button/index.js

```diff
@@ -83,6 +84,7 @@ export default function SaveButton( {
 			 */
 			showTooltip={ showTooltip }
 			icon={ icon }
+			__next40pxDefaultSize={ __next40pxDefaultSize }
```

Comment:
The `SaveButton` component of the edit-site package is also used in the header area of the editor. This button cannot be changed to 40px at the moment, so I have passed `__next40pxDefaultSize` as props.

![edit-site-save](https://github.com/WordPress/gutenberg/assets/54422211/9c685335-eb13-4e22-93b9-2c51b4423ad1)